system-test/storage.js

```diff
@@ -696,6 +696,25 @@ describe('storage', function() {
     });
   });
 
+  describe('channels', function() {
```

Comment:
Or is that not happening because of the difficulty involved in creating a channel..?